crates/turbopack-core/src/resolve/alias_map.rs

```diff
@@ -465,22 +470,25 @@ where
             for (key, template) in &mut *current_prefix_iterator {
                 match key {
                     AliasKey::Exact => {
-                        if self.request.len() == prefix.len() {
-                            return Some(AliasMatch::Exact(template));
+                        if self
+                            .request
+                            .as_string()
```

Comment:
The Pattern might not be convertable to a string, but it could still match an Exact alias